### PR TITLE
Be compatible with React 0.14, use "invariant" package

### PR DESCRIPTION
**Pull request by amireh**
What the title says! Not sure about the junk that was committed to `dist/` - I can remove if necessary.

### CODE REVIEW

File: ComponentInterpolator.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ var ComponentInterpolator = React.createClass({
   },
 
   interpolatePlaceholders(string) {
-    var token;
+    var token, child;
```
Comment (amireh) [inline ComponentInterpolator.js:98]: this was throwing an error when we assign to it that it is `undefined`

Comment (jenseng) [inline ComponentInterpolator.js:98]: oops, good catch

**Comment by jenseng** [comment]
thx man 👍  ... i'll publish a new version shortly

**Comment by amireh** [comment]
:+1:  Thank you!

**Pull request merged**